src/projected_distribution.jl

```diff
@@ -16,29 +16,27 @@ function ProjectedPointDistribution(M::Manifold, d::Distribution, proj!, x::TRes
     return ProjectedPointDistribution{TResult, typeof(M), typeof(d), typeof(proj!)}(M, d, proj!)
 end
 
-function support(d::ProjectedPointDistribution)
-    return MPointSupport(d.manifold)
-end
-
 function rand(rng::AbstractRNG, d::ProjectedPointDistribution{TResult}) where TResult
     x = convert(TResult, rand(rng, d.d))
     d.proj!(d.manifold, x)
     return x
 end
-
 function _rand!(rng::AbstractRNG, d::ProjectedPointDistribution, x::AbstractArray{<:Number})
     _rand!(rng, d.d, x)
     d.proj!(d.manifold, x)
     return x
 end
 
+function support(d::ProjectedPointDistribution)
+    return MPointSupport(d.manifold)
+end
 
 """
     ProjectedFVectorDistribution(type::VectorBundleFibers, x, d, project_vector!)
 
 Generates a random vector from ambient space of manifold `type.manifold`
 at point `x` and projects it to vector space of type `type` using function
-`project_vector!`.
+[`project_vector`](@ref).
```

Comment:
It uses the mutating variant so while the non-mutating one should have better docs, it's less precise.

Reply:
as for all other mutating ones, it actually oes not have a docstring itself (it also would up to 4-5 characters be a copy of the non-mutating docstring).

Reply:
What do you think about the following: We document somewhere quite prominently, that it is often preferable to use a mutating version, but the documentation always refers to the (then uniquely documented) non-mutating version? That's how we currently document, so I would prefer to continue that way – but yes the docs could mention this approach more prominently, maybe within the page for the interface.

Reply:
I was talking about this particular docstring for `ProjectedFVectorDistribution` -- it should say something like
```
projects it to vector space of type `type` using function `project_vector!`. See
[`project_vector`](@ref) for documentation.
```

I think everything else is fine.

Reply:
Ah, soryy, then I misunderstood this. I thought you wanted to have an explicit docstring for `project_vector!`(which would up to an additional signature parameter just be the copy of the nonmutating one), but you proposed change looks nice, it says explicitly, the function is using the mustating one, while one should check the docs of the non-mutating one. Nice solution, will add that tomorrow.